tests/ltp/patches/umount2_02.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/testcases/kernel/syscalls/umount2/umount2_02.c b/testcases/kernel/syscalls/umount2/umount2_02.c
+index 7d558fa11..256bc3f4d 100644
+--- a/testcases/kernel/syscalls/umount2/umount2_02.c
++++ b/testcases/kernel/syscalls/umount2/umount2_02.c
+@@ -26,6 +26,10 @@
+  *   MNT_DETACH. (fails with the error EINVAL)"
+  */
+ 
++/* Patch to use root file system for the test as loop device file
++ * system cannot be used as lkl Kernel Memory is set to 32M.
++ */
++
+ #include <errno.h>
+ #include <sys/mount.h>
+ 
+@@ -44,8 +48,8 @@ static void verify_failure(int i);
+ static void verify_success(int i);
+ static void cleanup(void);
+ 
+-static const char *device;
+-static const char *fs_type;
++static const char *device = "/dev/vda";
++static const char *fs_type = "ext4";
+ 
+ static int mount_flag;
+ 
+@@ -111,7 +115,7 @@ static void setup(void)
+ 	tst_sig(NOFORK, DEF_HANDLER, NULL);
+ 
+ 	tst_tmpdir();
+-
++/*
```

Comment:
Please remove the not needed code instead of commenting out. Cod eget messy with not needed and commented out code.

Reply:
will update